rodan-main/code/rodan/jobs/MEI_encoding/build_mei_file.py

```diff
@@ -539,6 +583,11 @@ def build_mei(pairs: List[Tuple[List[dict], dict]], classifier: dict, width_cont
         syl_dict = {"opening_syl": cur_syllable, "latest": syl, "added": False, "neume_added": False}
         # iterate over glyphs on the page that fall within the bounds of this syllable
         for i, glyph in enumerate(gs):
+            # if there are multiple columns, shift glyph box

```

Comment:
in comments, be clear on where you are shifting the glyph bounding boxes to and from. it's a complicated process, if someone needs to find a bug here in a year they're gonna have a hard time unless the comments are really explicit